src/platforms/stm32/src/lib/sys.c

```diff
@@ -263,8 +279,13 @@ Context *sys_create_port(GlobalContext *glb, const char *driver_name, term opts)
 
 term sys_get_info(Context *ctx, term key)
 {
-    UNUSED(ctx);
-    UNUSED(key);
+    GlobalContext *glb = ctx->global;
+    if (key == globalcontext_existing_term_from_atom_string(glb, ATOM_STR("\x12", "avm_free_heap_size"))) {
```

Comment:
We probably want this atom in defaultatoms.

Reply:
My understanding of the benefit of using `globalcontext_existing_term_from_atom_string()` is that we do not need to create the atom at all. Keeping the atom table smaller by not creating atoms that will never be returned to the user, since they are rarely used, and only as user input.